packages/sources/iex-cloud-test/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chainlink/iex-cloud-test-adapter",
+  "version": "0.0.0",
+  "description": "Chainlink iex-cloud-test adapter.",
+  "keywords": [
+    "Chainlink",
+    "LINK",
+    "blockchain",
+    "oracle",
+    "iex-cloud-test"
+  ],
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "url": "https://github.com/smartcontractkit/external-adapters-js",
+    "type": "git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "clean": "rm -rf dist && rm -f tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "tsc -b",
+    "server": "node -e 'require(\"./index.js\").server()'",
+    "server:dist": "node -e 'require(\"./dist/index.js\").server()'",
+    "start": "yarn server:dist"
+  },
+  "dependencies": {
+    "@chainlink/external-adapter-framework": "0.24.2",
```

Comment:
Even though this is the latest version, there's a new one coming soon. Adding this comment as a note to bump this version once that's out

Reply:
Version `0.25.0` is ready to be used

Reply:
Wouldn't be a complete review without another ask for a fw bump